traefik-image/traefik-image.kiwi.ini

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: _NAMESPACE_/traefik:%%LONG_VERSION%% _NAMESPACE_/traefik:%%LONG_VERSION%%-<RELEASE> -->
+
+<image schemaversion="6.5" name="_PRODUCT_-traefik-image">
+  <description type="system">
+    <author>SUSE Containers Team</author>
+    <contact>containers@suse.com</contact>
+    <specification>nginx ingress controller running on a _DISTRO_ container guest</specification>
```

Comment:
Sorry missed that on the first review, but the description needs to be updated :)

Reply:
sorry I missed that. thanks for the spotting it ;)